Automattic/newspack-blocks
# Donate Block: button color; remove Stripe badge

### All Submissions:






### Changes proposed in this Pull Request:



Change in the Donate block:

- changes the default CTA color to #dd3333 (red)
- makes this color customisable
- removes Stripe badge – it's taking some valuable screen space on mobile. Might be re-added at some point if user confidence in on-site checkout is an issue. 

### How to test the changes in this Pull Request:

1. Insert a Donate block
2. Observe it has red CTA color by default, in the editor and on the front-end
3. Observe the color can be updated
4. Observe no Stripe badge if Stripe is selected as the Reader Revenue platform

### Other information:
